src/api/courses.js

```diff
@@ -211,15 +211,55 @@ router.post(
   ],
   safeAsync(async (req, res, _next) => {
     const { name, shortcode } = matchedData(req)
+    const isUnlisted = req.body.isUnlisted
+    const questionFeedback = req.body.questionFeedback
```

Comment:
Would prefer `questionFeedbackEnabled` to match `queue.admissionControlEnabled`.